applications/app/controllers/OptInController.scala

```diff
@@ -26,7 +36,7 @@ object OptInController extends Controller {
     }))
   }
 
-  val HTTPS = OptInFeature("https_opt_in")
+  val HTTPS = HttpsOptFeature("https_opt_in")
```

Comment:
Fair comment. I ignored the casing of the variable name from the original I think as it made sense when I read it, being an abbreviation and all. I'll change it when I re-merge master in a few minutes :)